detekt.yml

```diff
@@ -95,7 +95,7 @@ complexity:
     excludeStringsWithLessThan5Characters: true
     ignoreStringsRegex: '$^'
   TooManyFunctions:
-    active: true
+    active: false
```

Comment:
I suggest keeping this enabled and maybe instead adjust the thresholds, but remember that it is enabled for a reason. Many functions in a file is often a good indication that a file is doing too much.

https://www.itamarweiss.com/code/2017/11/05/Clean-code-main-takeaways.html, https://softwareengineering.stackexchange.com/a/47517

Reply:
We should discuss this one in the call as well

Reply:
Done